README.md

```diff
@@ -29,8 +29,10 @@ elif os_is_centos; then
 fi
 ```
 
+#### Example of `dynamic-ubuntu-wait.sh` usage:
+Simply call the script by curling it during your existing provisioning/automated installation process:
 
-
+`curl -Ls https://raw.githubusercontent.com/gruntwork-io/bash-commons/dynamic-ubuntu-waiter/modules/bash-commons/src/dynamic-ubuntu-wait.sh | bash`
```

Comment:
`-LsS`. Also, that URL doesn't look quite right... And it should be versioned.

Reply:
We should come up with an official Gruntwork standard for `curl`. I actually like to use the fully laid out options of:

```
curl --location --silent --fail --show-error ...
```

This will:
- Follow redirects
- Output nothing
- ...unless there's an error
- Return a non-zero exit code if anything other than a 2XX HTTP response code is returned.

The one issue is that sometimes an API will return a 2XX response but along with an error, but I haven't been able to come up with anything better.

Reply:
Agreed for scripts, but with the `curl | bash` one-liner, you typically prefer brevity...

Reply:
Jim - is this what you were thinking for the url with version?

`curl -LsS https://raw.githubusercontent.com/gruntwork-io/bash-commons/[VERSION]/modules/bash-commons/src/dynamic-ubuntu-wait.sh | bash`

Reply:
Yup!

Reply:
In that case: done :-)

Reply:
Fair enough on the brevity, but I'm wondering if this should be:

```bash
curl -LsSf https://raw.githubusercontent.com/gruntwork-io/bash-commons/[VERSION]/modules/bash-commons/src/dynamic-ubuntu-wait.sh | bash
```

Try `curl`ing a URL that 404s and see if this returns a non-zero exit code (and therefore causes the entire script to exit), which is what we would want for an invalid URL. That's my goal in adding the `-f` / `--fail`. Otherwise, I'm worried that the `curl` call could fail (e.g. for transient reasons) and lead to confusing errors for users.